.gitlab-ci.yml

```diff
@@ -316,3 +316,31 @@ puppet-release:
   artifacts:
     paths:
       - deployments/puppet/pkg/*.tar.gz
+
+cve-scan:
+  image: ubuntu:20.04
+  stage: test
+  only:
+    - main
+    - schedules
+      - cron 0 8 * * *
+  script:
+    - make docker-otelcol
+    - mkdir -p ~/.docker/cli-plugins && \
+      curl https://github.com/docker/scan-cli-plugin/releases/latest/download/docker-scan_linux_amd64 -L -s -S -o ~/.docker/cli-plugins/docker-scan && \
+      chmod +x ~/.docker/cli-plugins/docker-scan
+    - yes | docker scan --login --token ${SNYK_AUTH_TOKEN}
+    - docker scan otelcol
+
+slack_message:
```

Comment:
We probably should call it something specific to CVEs failure like `notify-cve-scan-failure`.

Reply:
sure, I will change that